### ISSUE TITLE
Tooltip popups.

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#221]
I noticed suddenly help/tooltip icons. Which are very nice to have to add additional information and contain a link that goes directly to documentation related to the specific feature.

![Screenshot 2021-09-20 at 22 08 10](https://user-images.githubusercontent.com/5323259/134068280-454e4c97-4f92-41e1-90df-aa017e42c6df.jpg)

Clicking "Add another occurrence" tooltip ? .
![Screenshot 2021-09-20 at 22 09 01](https://user-images.githubusercontent.com/5323259/134068470-cb8d2fc4-7c1e-4468-81f1-db2723b4192b.jpg)

There is a lot of information in the bottom. What about removing the bottom line containing the links: My Calendar Help - Check out my other plugins, Rate this plugin 5 stars! and Help translate this plugin.

The most important link in this screen is Documentation. Which should link directly to the documentation containing screenshots and additional text related to the tooltip. Shortcode Generator and Get support are not that important in a tooltip popup screen which explains what a specific feature does.

Btw after clicking the x I noticed the following clickable area around the ?
![Screenshot 2021-09-20 at 22 09 12](https://user-images.githubusercontent.com/5323259/134068825-8cc811dd-1e2a-49d4-bc8c-d88363024386.jpg)

The bottom line of the clickable link should go a little further down.

**Comment by joedolson** [comment]
This is done; the link is a dummy, since the documentation isn't done, and will need to be updated later.

**Comment by paaljoachim** [comment]
Great!

**Comment by joedolson** [comment]
Closing; documentation link is no longer a dummy link.
